static/js/hashchange.js

```diff
@@ -22,6 +22,10 @@ import * as ui_util from "./ui_util";
 // or locally: docs/subsystems/hashchange-system.md
 let changing_hash = false;
 
+export function get_current_hash() {
+    return window.location.hash;
+}
+
```

Comment:
I don't think function is really the type of thing I have in mind -- it doesn't create a useful abstraction.  I'm OK with us deciding `subs.js` is too hard to clean up on this front, but it'd be better to have a function like `hash_util.is_create_streams_page()` that encodes this if we're trying to create a nice abstraction barrier between systems where we aren't parsing `window.location.hash` with slightly different logic everywhere.

Reply:
created is_create_new_stream_narrow in hashchange for this.